packages/patrol_cli/lib/src/features/drive/test_runner.dart

```diff
@@ -1,13 +1,33 @@
+import 'package:dispose_scope/dispose_scope.dart';
 import 'package:patrol_cli/src/features/drive/device.dart';
 
-typedef Test = Future<void> Function(Device device);
+typedef _Callback = Future<void> Function(String target, Device device);
```

Comment:
poor name

Reply:
it's private so I'd argue it's bearable

will think of a better name, and if not, I'll keep this in mind.